Assets/MirageXR/Common/Scripts/CombinedEditor/ActionDetailView/Annotation Editors/PickAndPlace/Pick.cs

```diff
@@ -91,6 +98,10 @@ void Update()
                 }
                 transform.hasChanged = false;
             }
+            else if(moveMode && pickOb.GetComponentInChildren<Renderer>().material.color != originalArrowColor)
```

Comment:
cache this value - `pickOb.GetComponentInChildren<Renderer>()` and use it instead.

Reply:
yeah, we should avoid is calling GetComponent every frame if we don't have to

Reply:
yeah, we should avoid is calling GetComponent every frame if we don't have to